stix2/test/test_environment.py

```diff
@@ -184,3 +184,25 @@ def test_parse_malware():
     assert mal.modified == FAKE_TIME
     assert mal.labels == ['ransomware']
     assert mal.name == "Cryptolocker"
+
```

Comment:
Add a test for when there is a configured data source, but that sources does not have the desired object.